bitnami/mongodb/values.yaml

```diff
@@ -189,6 +189,9 @@ tls:
   ## @param tls.enabled Enable MongoDB(&reg;) TLS support between nodes in the cluster as well as between mongo clients and nodes
   ##
   enabled: false
+  mTLS:
+    ## @param tls.mTLS.enabled IF TLS support is enabled, require clients to provide certificates
+    enabled: true
```

Comment:
Do you expect to extend mTLS configuration options in the future with further parameters? Otherwise, I'd rename this parameter to simply `mTLS` and treat it as a boolean.

Reply:
My rational was that I don't know for certain that there won't ever be any more fields related to mTLS, and I wanted to avoid setting up future breaking changes if I could avoid it. Off the top off my head, I don't know if MongoDB supports this, but by analogy, I know PostgreSQL has an option to have mtls enabled, but optional, vs enabled and required. If you feel this is overly cautious, I'll go ahead and change it.